WebApplication/Startup.cs

```diff
@@ -27,6 +27,7 @@
 using Microsoft.Extensions.Hosting;
```

Comment:
Maybe we should consider not to expose endopints at all. To make it obvious that this is separate application usecase.

Reply:
yes. I know. It is on my list. I have already tried it, but my changes did not worked.

Reply:
I will create another subtask for this.